enterprise/app/code/code.css

```diff
@@ -37,7 +37,7 @@
 
 .xrefs-container {
   overflow-y: auto;
-  height: 100%;
+  height: 85%;
```

Comment:
can you do this with `calc(100% - xxpx)` instead? this version makes the gap at the bottom change as the content is resized, yeah?

Reply:
Yeah, a calc is better - the xrefs-header is the competing element. I set the line height to 16px, and it has 20px of padding. `calc(100% - 36px)` avoids any cutoff.